test/langtools/tools/javac/diags/examples/ProbFoundReqFragment.java

```diff
@@ -22,7 +22,9 @@
  */
 
 // key: compiler.err.prob.found.req
-// key: compiler.misc.prob.found.req
+// key: compiler.misc.kindname.class
```

Comment:
Uhm - this sample is meant to test `compiler.misc.prob.found.req` (e.g. the fragment version). Perhaps we might need to look at the test framework that runs these diagnostic test to understand what we have to change - or what has changed - for this to work as before?